footstep_size_x: 0.24
footstep_size_y: 0.14
collision_bbox_size: [0.4, 1.2, 1.0]
collision_bbox_offset: [0, 0, 0]
successors:
  - x: 0
    y: -0.2
    theta: 0
  - x: 0
    y: -0.3
    theta: 0
  - x: 0
    y: -0.15
    theta: 0
  - x: 0.2
    y: -0.2
    theta: 0
  - x: 0.25
    y: -0.2
    theta: 0
  - x: 0.1
    y: -0.2
    theta: 0
  - x: -0.1
    y: -0.2
    theta: 0
  - x: 0
    y: -0.2
    theta: 0.17
  - x: 0
    y: -0.3
    theta: 0.17
  - x: 0.2
    y: -0.2
    theta: 0.17
  - x: 0.25
    y: -0.2
    theta: 0.17
  - x: 0.1
    y: -0.2
    theta: 0.17
  - x: 0
    y: -0.2
    theta: -0.17
  - x: 0
    y: -0.3
    theta: -0.17
  - x: 0.2
    y: -0.2
    theta: -0.17
  - x: 0.25
    y: -0.2
    theta: -0.17
  - x: 0.3
    y: -0.2
    theta: -0.17
  - x: 0.3
    y: -0.2
    theta: -0
  - x: 0.3
    y: -0.2
    theta: 0.17
  - x: 0.35
    y: -0.2
    theta: -0.17
  - x: 0.35
    y: -0.2
    theta: -0
  - x: 0.35
    y: -0.2
    theta: 0.17
  - x: 0.1
    y: -0.2
    theta: -0.17
  - x: 0.2
    y: -0.3
    theta: -0.17
  - x: 0.25
    y: -0.3
    theta: -0.17
  - x: 0.3
    y: -0.3
    theta: -0.17
  - x: 0.3
    y: -0.3
    theta: -0
  - x: 0.3
    y: -0.3
    theta: 0.17
  - x: 0.35
    y: -0.3
    theta: -0.17
  - x: 0.35
    y: -0.3
    theta: -0
  - x: 0.35
    y: -0.3
    theta: 0.17
